app/commands/user/destroy_account.rb

```diff
@@ -7,6 +7,8 @@ def call
     User::ResetAccount.(user)
     user.student_relationships.destroy_all
     user.mentor_relationships.destroy_all
+    user.subscriptions.update_all(user_id: User::GHOST_USER_ID)
+    user.payments.update_all(user_id: User::GHOST_USER_ID)
```

Comment:
The association sets the `:dependent` value to `:nullify`, but that is invalid as the `user_id` column is not nullable for payments or subscriptions. So we can either:

1. Make it nullable
2. Assign it to the ghost user

Reply:
I had that originally, but then saw that you have `nullify` explicitly defined so I didn't dare. Let me update